powerhome/playbook
# [BOP-1023] Add inline and inline compact options to the select kit for inline usage

#### Screens

<img width="1169" alt="Screenshot 2021-07-28 at 11 29 18 AM" src="https://user-images.githubusercontent.com/21108297/127377282-17a52f73-a299-485e-bc35-b7e7dd304d8e.png">
<img width="1167" alt="Screenshot 2021-07-28 at 11 29 23 AM" src="https://user-images.githubusercontent.com/21108297/127377286-8847f663-9baf-4cdb-a9ee-9190f2ad25f9.png">

#### Breaking Changes

None

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/BOP-1023

#### How to test this

Test from the Select Kit page, according to [this](https://pr234.playgrounds.beta.hq.powerapp.cloud/) prototype, it should be usable for both the Story/Bug change select and also the Sprint change select.

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
